docs/versionIndex.md

```diff
@@ -57,8 +57,8 @@ backend services have been tested with each released version of ADF.
 <!--v620 start-->
 
 -   [Search Properties component](content-services/components/search-properties.component.md)
--   [Search Date Range Advanced Component](content-services/components/search-date-range-advanced.component.md)
--   [Search Date Range Advanced Tabbed Component](content-services/components/search-date-range-advanced-tabbed.component.md)
+-   [Search Date Range Advanced Component](content-services/components/search-date-range.component.md)
```

Comment:
incorrect name probably as file is renamed to search-date-range.component.md so Advanced word should be removed